src/test/rustdoc/type-layout.rs

```diff
@@ -0,0 +1,46 @@
+// compile-flags: --show-type-layout -Z unstable-options
+
+// @has type_layout/struct.Foo.html 'Size: '
+// @has - ' bytes'
+pub struct Foo {
+    pub a: usize,
+    b: Vec<String>,
+}
+
+// @has type_layout/enum.Bar.html 'Size: '
+// @has - ' bytes'
+pub enum Bar<'a> {
+    A(String),
```

Comment:
Can we show the size of each enum field?

Reply:
This PR is already somewhat large, so I think we should keep this feature small initially and then can add to it over time.